src/components/Table/Table.stories.tsx

```diff
@@ -19,22 +21,41 @@ const Template = args => {
   return <Table {...args} />;
 };
 
+const SortableTableTemplate = args => {
+  const [tableStatus, setTableStatus] = useState({
```

Comment:
@viniciusaa not required (optional): I'd suggest adding a hook for these tables, and then we could have these generic states there, avoiding duplicating it for all projects

Reply:
Hey @pvieiraguimaraes , could you give me an idea on how to do this?
We could move this `useState` to inside the component, but how the user would access the `tableStatus` information?

Reply:
The latest update to the component simplifies its usage. We can now manipulate the sorting data using just one prop (`setSortState`). Could you please review this and provide your feedback? If required I can revert these last changes.